### ISSUE TITLE
Long Reference Strip Scroll significantly affects performance

### ISSUE THREAD

**Issue by DominiqueDeCordova** [openseadragon/openseadragon#2128]
When I have several hundred images in reference strip scroll, the **UI becomes unresponsive**.
The code looks like:
```
sequenceMode: true,
showReferenceStrip: true,
referenceStripScroll: 'vertical',
tileSources: [{
            type: 'legacy-image-pyramid',
            levels: [{
                url: 'http://openseadragon.github.io/example-images/rbc/rbc0001/2003/2003rosen1799/0001q.jpg',
                height: 889,
                width:  600
            }]
        },{
            type: 'legacy-image-pyramid',
            levels: [{
                url: 'http://openseadragon.github.io/example-images/rbc/rbc0001/2003/2003rosen1799/0002q.jpg',
                height: 889,
                width:  600
            }]
        } ... ]
```
Please advise.

**Comment by shyamkumaryadav** [comment]
see #2108

**Comment by kriewall** [comment]
@iangilman Is it possible to implement paging in the reference strip such that only the images in view (or nearly in view) are loaded?

**Comment by iangilman** [comment]
@kriewall Yes, that seems like an excellent solution! I believe each of the elements in the reference strip are actually little OSD viewers, so there's probably a way to tell them to hide/show themselves. 

Of course, having them be OSD viewers is probably overkill; it would probably be fine for them to be simple images, but then somewhere we would need to figure out the logic for making sure they came from the right tile(s) or whatever. That would be a much bigger refactor. 

Anyway, if you wanted to try doing some paging for the reference strip, that would be great! If you're up for making a pull request with it, even better :)

**Comment by kriewall** [comment]
@iangilman My counterpart @DominiqueDeCordova will take a crack at it.  Javascript is not my forte, so you probably don't want me mucking around in there.  :)

**Comment by iangilman** [comment]
Sounds good! @DominiqueDeCordova Let me know if you have any questions!
